packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -477,6 +477,7 @@ describe('<Tooltip />', () => {
       });
 
       expect(getByRole('tooltip')).toBeVisible();
+      expect(handleFocus.callCount).to.equal(1);
```

Comment:
To match the test description "should handle autoFocus + onFocus forwarding". The "+ onFocus forwarding" part was not tested as far as I can tell.